addons/necropost-highlighter/addon.json

```diff
@@ -0,0 +1,174 @@
+{
+  "name": "Necropost highlighter",
```

Comment:
```suggestion
  "name": "Forums necropost highlighter",
```

Reply:
I only see the term Necropost used about posts on the Scratch site, so "forum" is implied.

Most other addons that apply to the forums don't begin with the word Forum (some don't use it at all). If the word "forum" is required, I'd prefer:

Necropost highlighter (forums)

or the less scannable

Necropost highlighter on forums